lib/index.js

```diff
@@ -172,16 +192,15 @@ module.exports = class DockerComposeParser {
 
     return validate(serviceSchema, rawService)
     // Map raw properties
-    .then(() => {
-      rawService.environment = Mapper.mapKeyValueObjectToArray(rawService.environment)
-      return rawService
-    })
-    // Populate Warnings
-    .then(rawService => Warning.unsupportedKeys(rawService, warnings, serviceSchemaKeys))
-    .then(rawService => Warning.buildAndImage(rawService, warnings))
```

Comment:
Why did we remove this? If we remove it, we should also remove all the test for this, but not sure we should.